modulefiles/ufs_gaea.intel

```diff
@@ -25,12 +25,12 @@ module load gcc/8.3.0
 module use /lustre/f2/pdata/ncep_shared/hpc-stack/modulefiles/stack
 
 module load hpc/1.1.0
-module load hpc-intel/18.0.6.288
+module load hpc-intel/2021.3.0
 module load hpc-cray-mpich/7.7.11
 
 module load ufs_common
 
-setenv I_MPI_DAPL_UD 1
+#setenv I_MPI_DAPL_UD 1
```

Comment:
If this is not needed anymore I would delete this line

Reply:
removed.